src/ripple/app/misc/AmendmentTable.h

```diff
@@ -272,22 +272,25 @@ class AmendmentTable
         // Inject appropriate pseudo-transactions
         for (auto const& it : actions)
         {
-            STTx trans (ttAMENDMENT);
-            trans.setAccountID (sfAccount, AccountID());
-            trans.setFieldH256 (sfAmendment, it.first);
-            trans.setFieldU32 (sfLedgerSequence, lastClosedLedger->seq() + 1);
+            STTx amendTx (ttAMENDMENT,
+                [&it, seq = lastClosedLedger->seq() + 1](STObject& obj)
```

Comment:
In Effective Modern C++ Item 31, Meyers recommends avoiding default capture modes (both by-value and by-reference).  His rationale is that they can lead to dangling references.  I wouldn't say "never" use default capture modes.  But without motivation to do otherwise, this one seems pretty reasonable to me.